src/components/FindMovie/FindMovie.tsx

```diff
@@ -64,6 +64,8 @@ export const FindMovie: FC<Props> = ({ addMovie }) => {
       });
   };
 
+  const cn = className(isError ? 'input is-danger' : 'input');
```

Comment:
this is not what I meant)

you should write like this
```
className('input', {
  'is-danger': isError,
})
```

Reply:
and you can move this back to attribute

Reply:
it's a fiasco) now I understand, thanks

Reply:
it's just another experience))